otter/log/formatters.py

```diff
@@ -332,11 +244,8 @@ def Observer(eventDict):
             if key not in PRIMITIVE_FIELDS:
                 log_params[key] = value
 
-        # emit an audit log entry also, if it's an audit log
         if 'audit_log' in eventDict:
```

Comment:
There is one dashboard specifically of audit log feeds, which I changed to use this one.  I'm not sure if it's used by support at all atm.  Also we have way too many dashboards to even list, so I can't be sure I got all of them.